docs/book/src/quick-start.md

```diff
@@ -37,6 +37,9 @@ export PATH=$PATH:/usr/local/kubebuilder/bin
 
 Also, you can install a master snapshot from `https://go.kubebuilder.io/dl/latest/${os}/${arch}`.
 
+Optional: To enable kubebuilder auto completion, please [refer][./refernce/completion.md]
+
+
```

Comment:
It is not regarded to use the master branch and it is not specific for the master. 
IMO it would be a specific note as well. 

<img width="835" alt="Screenshot 2020-03-26 at 11 14 49" src="https://user-images.githubusercontent.com/7708031/77641028-44797280-6f53-11ea-971a-0afe52a05bcc.png">

Also, see that the. link is not appearing correctly. To check how to preview the doc changes see: https://github.com/kubernetes-sigs/kubebuilder/blob/master/CONTRIBUTING.md#how-to-preview-the-changes-performed-in-the-docs